packages/zipkin-transport-http/src/HttpLogger.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-console */
+const fetch = require('node-fetch');
+
+class HttpLogger {
+  constructor({endpoint, httpInterval = 1000}) {
+    this.endpoint = endpoint;
+    this.queue = [];
+
+    setInterval(() => {
+      this.processQueue();
+    }, httpInterval);
+  }
+
+  logSpan(span) {
+    this.queue.push(span.toJSON());
+  }
+
+  processQueue() {
+    if (this.queue.length > 0) {
+      const postBody = JSON.stringify(this.queue);
+      fetch(this.endpoint, {
+        method: 'POST',
+        body: postBody,
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json'
+        }
+      }).then((response) => {
+        if (response.status === 202) {
+          this.queue.length = 0;
```

Comment:
What do you think about truncating the queue in every case, regardless of whether the request succeeds or not? We don't require consistency in Zipkin trace data. My concern is that if the zipkin server is down, the posting would fail, and this.queue would never be emptied, leading to a memory leak.